src/index.jsx

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] Kindly don't disable your linters and try to fix your linters manually 👍 because linters prevents your from introducing those errors which can break your app 😨